dcs/src/test/jdbc_test/src/test/java/org/trafodion/jdbc_test/TestGetIndexInfo.java

```diff
@@ -0,0 +1,196 @@
+			ResultSet indexInformation = meta.getIndexInfo(_conn.getCatalog(), null, INDEX_INFO_TEST_TABLE, false, false);
+			
+			int rowNum = 0;
+			while(indexInformation.next()) {
+				compareInfoWithExp("testGetUniqueIndexInfo", rowNum + 1, indexInformation, expIndexInfo[rowNum]);
+				rowNum += 1;
+			}
+			assertEquals(rowNum, 3);
+		} catch(Exception e) {
+			System.out.println(e.getMessage());
+			e.printStackTrace();
+		}
+    }
+
+	@AfterClass
+    public static void cleanTable() {
+    	try ( Statement stmt = _conn.createStatement() ) {
+    		stmt.execute(strDropTableQuery);
+    	} catch(Exception e) {
+    		// do nothing
+    	}
+    	
+    	try {
+    		_conn.close();
+    	} catch(Exception e) {
+    		
+    	}
+    }
+	
+	private class IndexInfo {
+		public String dbCatalog = "TRAFODION";
+		public String dbSchema = "SEABASE";
+		public String dbTableName;
+		public boolean dbNoneUnique = false;
+	    public String dbIndexQualifier;
+	    public String dbIndexName;
+	    public short dbType;
+	    public short dbOrdinalPosition;
+	    public String dbColumnName;
+	    public String dbAscOrDesc;
+	    public int dbCardinality;
+	    public int dbPages;
+	    public String dbFilterCondition;
+	    
+	    public IndexInfo(String dbCatalog, String dbSchema, String dbTableName, boolean dbNoneUnique, String dbIndexQualifier, String dbIndexName,
+	    		short dbType, short dbOrdinalPosition, String dbColumnName, int dbAscOrdesc, int dbCardinality, int dbPages, String dbFilterCondition) {
+	    	this.dbCatalog = dbCatalog;
+	    	this.dbSchema = dbSchema;
+	    	this.dbTableName = dbTableName;
+	    	this.dbNoneUnique = dbNoneUnique;
+	    	this.dbIndexQualifier = dbIndexQualifier;
+	    	this.dbIndexName = dbIndexName;
+	    	this.dbType = dbType;
+	    	this.dbOrdinalPosition = dbOrdinalPosition;
+	    	this.dbColumnName = dbColumnName;
+	    	this.dbCardinality = dbCardinality;
+	    	this.dbPages = dbPages;
+	    	this.dbFilterCondition = dbFilterCondition;
+	    }
+	}
+	
+	private void compareInfoWithExp(String methondName, int rowNum, ResultSet rs, IndexInfo indexInfo) {
+		try {
+			assertEquals(methondName + " rowNum " + Integer.toString(rowNum) + " dbCatalog ", indexInfo.dbCatalog, rs.getString("TABLE_CAT"));
+			String currentSchema = _conn.getSchema();
+			
+			// Since we may run the test in different Schema, so compare the result to the current schema of the connection here.
+			assertEquals(methondName + " rowNUm " + Integer.toString(rowNum) + " dbSchem ", currentSchema, rs.getString("TABLE_SCHEM"));
+			
+			assertEquals(methondName + " rowNum " + Integer.toString(rowNum) + " dbTableName ", indexInfo.dbTableName, rs.getString("TABLE_NAME"));
+			assertEquals(methondName + " rowNum " + Integer.toString(rowNum) + " dbNoneUnique ", indexInfo.dbNoneUnique, rs.getBoolean("NON_UNIQUE"));
+			// By the Document, dbNoneUnique will return null if the type is SQL_TABLE_STAT
+			System.out.println(rs.wasNull());

```

Comment:
Is it needed to display the wasNull.  Where will the output go in Jenkins environment